openapi-core/src/main/java/io/ballerina/openapi/core/generators/client/parameter/QueryParameterGenerator.java

```diff
@@ -145,9 +147,14 @@ public List<ClientDiagnostic> getDiagnostics() {
         return diagnostics;
     }
 
-    private boolean isQueryParamTypeSupported(SyntaxKind type) {
-        return type.equals(BOOLEAN_TYPE_DESC) || type.equals(INT_TYPE_DESC) || type.equals(FLOAT_TYPE_DESC) ||
-                type.equals(DECIMAL_TYPE_DESC) ||
-                type.equals(STRING_TYPE_DESC) || type.equals(MAP_TYPE_DESC) || type.equals(ENUM_DECLARATION) || type.equals(ARRAY_TYPE_DESC);
+//    private boolean isQueryParamTypeSupported(SyntaxKind type) {
+//        return type.equals("boolean") || type.equals("integer") || type.equals("number") ||
+//                type.equals("array") ||
+//                type.equals("string") || type.equals(MAP_TYPE_DESC) || type.equals(ENUM_DECLARATION) || type.equals(ARRAY_TYPE_DESC);
+//    }
```

Comment:
Do we need to keep this commented code?

Reply:
Currently, we are working on a separate branch these codes for on demand dependable fix for wip revamp, this will be removed once the PR is finalised :)